pkg/sync/v5/sync.go

```diff
@@ -300,22 +300,27 @@ func (s *sync) writeDB() error {
 	sort.Strings(keys)
 
 	// crd needs to land early to get initialized
+	s.log.Debug("applying crd resources")
```

Comment:
if adding these, make sure we don't run at level debug by default - we were way too noisy and I don't want to go back there

Reply:
Increased default level to Info